src/js/media-captions-menu-button.js

```diff
@@ -0,0 +1,161 @@
+import './media-chrome-menu-button.js';
+import './media-captions-listbox.js';
+import { MediaUIAttributes, MediaStateReceiverAttributes } from './constants.js';
+import { defineCustomElement } from './utils/defineCustomElement.js';
+import {
+  Window as window,
+  Document as document,
+} from './utils/server-safe-globals.js';
+
+const template = document.createElement('template');
+
+template.innerHTML = `
+<style>
+  :host,
+  media-chrome-menu-button {
+    display: contents;
+  }
+
+  media-captions-listbox {
+    position: absolute;
+    bottom: 44px;
+    margin-left: -34px;
+    max-height: 300px;
+    overflow: auto;
+  }
+
+  /*
+   * if no captions selected, enabled icon should be hidden
+   * if captions selected, the disabled icon should be hidden
+   */
+  media-chrome-menu-button:not(.enabled) .enabled,
+  media-chrome-menu-button.enabled .disabled {
+    display: none;
+  }
+  /*
+   * if captions selected, enabled icon should be shown
+   * if no captions selected, disabled icon should be shown
+   */
+  media-chrome-menu-button.enabled .enabled {
+  media-chrome-menu-button:not(.enabled) .disabled {
+    display: initial;
+  }
+</style>
+
+<media-chrome-menu-button aria-label="captions menu button">
+  <svg class="enabled" slot="button-content" aria-hidden="true" viewBox="0 0 26 24">
```

Comment:
nitpick(non-blocking): Let's break these svgs out into their own strings to conform with our tacit convention in other buttons.